### ISSUE TITLE
new module: LDSC/Genetic Correlation

### ISSUE THREAD

**Issue by juliaapolonio** [nf-core/modules#5205]
Create a module to run the genetic correlation analysis on [LDSC](https://github.com/bulik/ldsc) tool

### Is there an existing module for this?

- [X] I have searched for the existing module

### Is there an open PR for this?

- [X] I have searched for existing PRs

### Is there an open issue for this?

- [X] I have searched for existing issues

### Are you going to work on this?

- [X] If I'm planning to work on this module, I added myself to the `Assignees` to facilitate tracking who is working on the module

**Comment by famosab** [comment]
@juliaapolonio Is this still planned? :)

**Comment by juliaapolonio** [comment]
Hi! This is not my priority for this year's hackathon. Should I close this issue?

**Comment by famosab** [comment]
Yes if you do not plan to work on it you can close it :) If someone else picks it up they can reopen!
